h/static/scripts/group-forms/components/forms/Checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import type { CheckboxProps as BaseCheckboxProps } from '@hypothesis/frontend-shared';
+import { Checkbox as BaseCheckbox } from '@hypothesis/frontend-shared';
+import type { ComponentChildren } from 'preact';
+import { useId } from 'preact/hooks';
+
+export type CheckboxProps = Omit<BaseCheckboxProps, 'aria-describedby'> & {
+  /**
+   * Adds a description right under the checkbox and main content, aligned with
+   * the main content's left side.
+   */
+  description?: ComponentChildren;
+
+  'aria-describedby'?: string;
+};
+
+export default function Checkbox({
+  description,
+  children,
+  'aria-describedby': ariaDescribedBy,
+  ...checkboxProps
+}: CheckboxProps) {
+  const descriptionId = useId();
+
+  const describedBy: string[] = [];
+  if (ariaDescribedBy) {
+    describedBy.unshift(ariaDescribedBy);
+  }
+  if (description) {
+    describedBy.push(descriptionId);
+  }
+
+  return (
+    <div>
+      <BaseCheckbox {...checkboxProps} aria-describedby={describedBy.join(' ')}>
+        <span className="text-grey-7">{children}</span>
+      </BaseCheckbox>
+      {description && (
+        <div
+          data-testid="description"
+          id={descriptionId}
+          className="text-grey-6 ml-5"
+        >
+          {description}
+        </div>
```

Comment:
I think this could do with a little more vertical room between the checkbox label and description.

Reply:
Yep, makes sense. In the radio buttons we set `mt-1` in the descriptions. I'll do the same here.